vyper/parser/constants.py

```diff
@@ -73,10 +73,10 @@ def unroll_constant(self, const, global_ctx):
 
         if fail:
             raise TypeMismatchException(
-                'Invalid value for constant type, expected %r got %r instead' % (
-                    annotation_type,
-                    expr.typ,
-                ),
+                f"""
+                Invalid value for constant type, expected {annotation_type} got
+                {expr.typ} instead'
+                """,
```

Comment:
Changing this a multi-line string will result in some confusing formatting, it might be better to split it over two single-line strings.

Reply:
I hadn't thought of this at the time - it also let me get rid of most of the `noqa: E501`s that I'd introduced.